### DISCUSSION TITLE
Improve “Relations” reference the graphql equivalent in the docs

### DISCUSSION THREAD

**Discussion by toyflish** [craftcms/docs#306] [Ideas]
I have a suggestion for https://craftcms.com/docs/3.x/relations.html
there is no documentation on how to query entries with relatedTo on fields that are in a matrix block.
Would be nice to have cross references in the docs between the query syntax and the graphql equivalent.

**Comment by mattstein** [comment]
Thanks for both suggestions, @toyflish!
